sensor/docs/tutorials/tag_based_pnp_calibrator.md

```diff
@@ -0,0 +1,153 @@
+# tag_based_pnp_calibrator
+
+In this tutorial, we present a hands-on tutorial of the `tag_based_pnp_calibrator`. Although we provide pre-recorded rosbag, the flow of the tutorial is meant to show the user the steps they must perform in their own use cases with live sensors.
+
+General documentation regarding this calibrator can be found [here](../../tag_based_pnp_calibrator/README.md).
+
+## Setup
+
+This tutorial assumes that the user has already built the calibration tools.
+Installation instructions can be found [here](../../README.md)
+
+## Data preparation
+
+Please download the data (rosbag) from [here](https://drive.google.com/drive/folders/1gFrrchW9mWM1huWMYuJ0nWB2n1BfCJag).
+
+The provided rosbag includes four different topics `camera_info`, `image_rect_color/compressed`, `pointcloud_raw`, and `tf_static`.
+
+## Environment preparation
+
+### Overall calibration environment
+
+The required space for calibration depends on the vehicle and sensors used. During the calibration, please always make sure that tags are detected by both camera and lidar. The user could always check whether the tags appear in the `rviz` and the `image view` UI.
+
+### AprilTag (LidarTag)
+
+Depending on the lidar model and the available space, the required AprilTag size may vary, but so far we have had good results with 0.6m and 0.8m tags (the provided sizes correspond to an edge's size. In these cases the payloads are 0.45m and 0.6m).
+
+In addition to the size of the AprilTag, which affects the physical positions where a tag can be detected, the structure on which the AprilTag is mounted is equally important. The shape and size of this structure may affect the lidar detection algorithm. Therefore, it is advisable to use a mount that positions the tag in such a way that it remains invisible to the sensor (refer to the provided example).
```

Comment:
The provided example in the main documentation, so it is not clear to what you refer here.
The images in this page do not show well the mount, but if you want to use them, please refer to those with some indexing

Reply:
Thanks!
I removed them in https://github.com/tier4/CalibrationTools/pull/164/commits/cc75c68d22709c28e96567be1b5d47320469fb34.

Reply:
` (so far, there have been no issues).`
This is not a useful explanation.
It could be: "so far, in our use cases, the tag is always detected as long as it is completely visible in the image"

Additionally, `Information can be found`
is not quite narutal:
`Details can be found` is better in my opinion

Reply:
Thanks, fixed in https://github.com/tier4/CalibrationTools/pull/164/commits/fd1c496905d40c494cfbf55c09b962dd863aac0e